charts/ocis/values.yaml

```
@@ -1514,6 +1514,12 @@ services:
         path: ""
       # URI where to redirect the user after a logout was performed. Defaults to the URI of the login page.
       postLogoutRedirectURI: ""
+      # Specifies the target URL valid for the logged out / access denied page.
```

```suggestion
      # Specifies the target url valid for the logged out / access denied page.
```
Either the url here should be written lower case or the url in the other descriptions should be written capital case.